src/app/components/UsefulLinks/index.tsx

```diff
@@ -0,0 +1,64 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import { Summary } from '#app/models/types/curationData';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import Heading from '../Heading';
+import styles from './index.styles';
+
+interface UsefulLinksProps {
+  id?: string;
+  title: string;
+  summaries: Summary[];
+}
+
+const UsefulLinks = ({
+  title,
+  summaries = [],
+  id = 'useful-links-1',
+}: UsefulLinksProps) => {
+  const { dir } = useContext(ServiceContext);
+
+  if (!summaries.length) {
+    return null;
+  }
+
+  const hasMultipleSummaries = summaries.length > 1;
+
+  return (
+    <section
+      role="region"
+      aria-labelledby={id}
+      data-testid={id}
+      css={styles.container}
+    >
+      <Heading level={2} id={id} css={styles.heading} dir={dir}>
+        {title}
+      </Heading>
+      {hasMultipleSummaries ? (
+        <ul css={styles.unorderedList} dir={dir} role="list">
+          {summaries.map(summary => {
+            return (
+              <li css={styles.item} key={summary.title}>
+                <a href={summary.link} css={styles.link}>
+                  {/* eslint-disable-next-line jsx-a11y/aria-role */}
+                  <span role="text">{summary.title}</span>
```

Comment:
`<span` role="text">` should not be needed, this is only needed to mitigate text splitting by screen readers if there is more than one nested element.